catalog/src/main/java/org/killbill/billing/catalog/StandaloneCatalog.java

```diff
@@ -161,7 +162,7 @@ public boolean isTemplateCatalog() {
                (supportedCurrencies == null || supportedCurrencies.length == 0);
     }
 
-    public DefaultPlanRules getPlanRules() {
+    public PlanRules getPlanRules() {
```

Comment:
This forces a cast in `DefaultMutableStaticCatalog` - worth the change?

Reply:
Yep, i have no choice as this is now a `StaticCatalog` api. However i am missing the `@Override` annotation that would have made that point more clear. Will add it.